mmv1/third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -766,6 +814,7 @@ resource "google_bigquery_dataset" "test" {
 }
 
 resource "google_bigquery_table" "test" {
+	deletion_protection = false
```

Comment:
Same as the examples above, tf config strings are typically 2 spaces instead of tabs.